### ISSUE TITLE
autoEdit 2 text in nav displays as link but does not direct anywhere

### ISSUE THREAD

**Issue by jhdouglas** [OpenNewsLabs/autoEdit_2#5]
Upon opening the app, users are brought to a landing screen that lists transcripts if they are available. In the nav are links view transcripts, add new files or view help options. Also included is “autoEdit 2” text. This text currently displays as a link when moused over but does not direct users anywhere.

1) Open autoEdit 2 app
2) View options in nav
3) Hover over “autoEdit 2” text in nav
4) Observe text displays as link
5) Click text
6) Note link does not direct anywhere

<img width="799" alt="screenshot 2016-12-09 10 23 13" src="https://cloud.githubusercontent.com/assets/5640831/21054272/ed969c20-bdfa-11e6-98d1-48e9f9462230.png">

**Comment by jhdouglas** [comment]
Looks like this actually directs users to the list of transcripts when clicked. Was not apparent when already viewing that screen. Please close if this is by design.

**Comment by pietrop** [comment]
For now added code to disable brand  in `nwjs/custom.css`

```
/* disabling link for brand*/
#brandAppName{
 pointer-events: none;
 cursor: default;
}
```

Fixed it for version `1.0.6`.
